project/Mima.scala

```diff
@@ -27,33 +27,35 @@ object Mima {
   }
 
   val apiCompatibilityExceptions: Seq[ProblemFilter] = Seq(
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.apply"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.copy"),
-    ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.meta.Dialect.copy$default$21"),
-    ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.meta.Dialect.copy$default$22"),
-    ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.meta.Dialect.copy$default$23"),
-    ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.meta.Dialect.copy$default$24"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.copy$default$22"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.copy$default$23"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.copy$default$24"),
-    ProblemFilters
-      .exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowImplicitFunctionTypes"),
-    ProblemFilters
-      .exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowImplicitFunctionTypes"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowMethodTypes"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowMethodTypes"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowWithTypes"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowWithTypes"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowWhiteboxMacro"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowWhiteboxMacro"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowLiteralUnitType"),
-    ProblemFilters
-      .exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowLiteralUnitType"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowSuperTrait"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowSuperTrait"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.allowAsPatternBinding"),
-    ProblemFilters
-      .exclude[DirectMissingMethodProblem]("scala.meta.Dialect.withAllowAsPatternBinding"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.Dialect.this")
+    ProblemFilters.exclude[MissingClassProblem]("scala.meta.Type$And"),
```

Comment:
Could we leave the trees for now? Just add the deprecated message next to them. We can remove them later.

Reply:
done